src/ping.ts

```diff
@@ -0,0 +1,233 @@
+const completionSpec: Fig.Spec = {
+  name: "ping",
+  args: {
+    name: "Hostname or IP",
+  },
+  description: "Send ICMP ECHO_REQUEST to network hosts",
+  options: [
+    {
+      name: "-A",
+      description:
+        "Audible.  Output a bell (ASCII 0x07) character when no packet is received before the next packet is transmitted.  To cater for round-trip times that are longer than the interval between trans- missions, further missing packets cause a bell only if the maxi- mum number of unreceived packets has increased",
```

Comment:
Extra space after the `.`. Looks like this issue occurs multiple times, could you fix this in the whole file?
```suggestion
        "Audible. Output a bell (ASCII 0x07) character when no packet is received before the next packet is transmitted. To cater for round-trip times that are longer than the interval between trans- missions, further missing packets cause a bell only if the maxi- mum number of unreceived packets has increased",
```